idf_component_manager/cli.py

```diff
@@ -116,7 +116,10 @@ def remove_managed_components(manager):
     manager.remove_managed_components()
 
 
-MANIFEST_COMPONENT_NAME_OPTION = [click.option('--component', default='main', help='Component name in the project')]
+MANIFEST_OPTIONS = [
+    click.option('--component', default='main', help='Component name in the project'),
+    click.option('--component-path', default=None, help='Component path in the project'),
```

Comment:
I suggest naming this option `['-p', '--path']` to make it similar to other `idf.py` commands like `create-project`.

Reply:
It's not clear if the path is relative or absolute. Implementation in this PR can accept also absolute paths outside of the project's directory. It is still can be useful, just need to be specified.

Reply:
You're absolutely right about the relative/absolute confusion. I see two other similar implementations in the ComponentManager [`__init__`](https://github.com/espressif/idf-component-manager/blob/d86fdae00ec4b159ce7f260d41b063d027b69059/idf_component_manager/core.py#L72) and [`create_project_from_example`](https://github.com/espressif/idf-component-manager/blob/d86fdae00ec4b159ce7f260d41b063d027b69059/idf_component_manager/core.py#L133) where path is absolute. Would it make sense to follow this pattern here? I would also add a check to make sure path is a directory.

Reply:
I renamed the option, raised an error if component and path are passed, and checked if path is under project path.